src/harness/client.ts

```diff
@@ -772,8 +772,10 @@ namespace ts.server {
             return notImplemented();
         }
 
-        getEncodedSemanticClassifications(_fileName: string, _span: TextSpan, _format?: SemanticClassificationFormat): Classifications {
-            return notImplemented();
+        getEncodedSemanticClassifications(file: string, span: TextSpan, format?: SemanticClassificationFormat): Classifications {
+            const request = this.processRequest<protocol.EncodedSemanticClassificationsRequest>(protocol.CommandTypes.EncodedSemanticClassificationsFull, { file, start: span.start, length: span.length, format });
+            const r = this.processResponse<protocol.EncodedSemanticClassificationsResponse>(request);
+            return r.body!;
         }

```

Comment:
This is the actual call which vscode uses, so I've replicated that codepath for the server tests. The non-encoded versions of the APIs aren't in the server, and it doesn't feel right to add them just for a test.